src/accordion/accordion.js

```diff
@@ -79,7 +79,9 @@ angular.module('ui.bootstrap.accordion', ['ui.bootstrap.collapse'])
     link: function(scope, element, attrs, accordionCtrl) {
       accordionCtrl.addGroup(scope);
 
+      scope.openClass = attrs.openClass || 'panel-open';
       scope.$watch('isOpen', function(value) {
+        element.toggleClass(scope.openClass, value);
```

Comment:
I apologize, I'm on not enough sleep it appears - this is completely fine due to the second parameter. Ignore my complaints here.